Ejercicio 4 POO/Batalla.ts

```diff
@@ -0,0 +1,32 @@
+import { Personaje } from "./Personaje";
+export class Batalla{
+    private jugadorUno:Personaje;
+    private jugadorDos:Personaje;
+    constructor(jugadorUno:Personaje,jugadorDos:Personaje){
+        this.jugadorUno = jugadorUno;
+        this.jugadorDos = jugadorDos;
+    }
+    public pelear(jugadorUno:Personaje, jugadorDos:Personaje){
```

Comment:
Sería innecesario que pelear reciboa los jugadores por parámetro ya que ya los recibe mediante el constructor.

Cuando se instancia la batalla se pasan los jugadores en el constructor para darle valor a las variables `jugadorUno `y `jugadorDos`.
Y cuando necesites llamarlos desde pelear lo haces con el `this.variable`